java-chassis-reference/en_US/start/development-environment.md

```diff
@@ -1,24 +1,24 @@
-# 安装本地开发环境
+# Prepare the local development environment
 
-应用开发环境所需安装的工具包括JDK、Maven、Eclipse 和 IDEA 。如果已经安装了这些开发工具，请跳过本节。
+The tools required to install the development environment include JDK, Maven, Eclipse, and IDEA. If you already have these development tools installed, skip this section.
```

Comment:
The tools required to install the development environment include JDK, Maven, Eclipse, and IDEA ->
JDK, Maven, Eclipse, and IDEA is required.